apps/judicial-system/README.md

```diff
@@ -274,7 +274,7 @@ Then you need to update the Helm charts. Add `SECRET_FEATURE` to `HIDDEN_FEATURE
 Then run the script
 
 ```
-./infra/scripts/generate-chart-values.sh judicial-system
+yarn charts
```

Comment:
This is missing the judicial-system namespace?
```suggestion
yarn charts judicial-system
```

Reply:
The sh script did not accept any parameters previously (probably because of a refactor and no-one updated the readme)